app/api_contract.py

```diff
@@ -0,0 +1,385 @@
+import datetime
+import enum
+import uuid
+from typing import Annotated
+
+from fastapi import FastAPI, Header, status
+from pydantic import BaseModel, EmailStr, Field, JsonValue
+
+app = FastAPI()
+
+
+JWTToken = Annotated[
+    str,
+    Header(
+        alias="Authroization",
+        title="JWT Token",
+        description="The JWT token used for authorization in `Bearer <token>` format",
+    ),
+]
```

Comment:
As mentioned in the PR description, this is just for documentation purposes

Not sure if that's how we'll implement providing the JWT token, what are your thoughts?